tests/acceptance/features/coreApiWebdavProperties/copyFile.feature

```diff
@@ -815,6 +857,41 @@ Feature: copy file
       | old              |
       | new              |
 
+
+  Scenario Outline: copying a file between shares received from different users when one share is shared via Viewer and Secure viewer permission
```

Comment:
```suggestion
  Scenario Outline: copying a file between shares received from different users when one share is shared via Viewer and Secure viewer permission
```

In just above scenario, it is written as `secure viewer` and here in this scenario it is `Secure viewer`. Just casing things. :grin:  Can we please make those same in all scenarios ensuring consistency?